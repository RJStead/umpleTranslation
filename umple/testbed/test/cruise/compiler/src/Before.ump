generate Java "../src-gen-umple";

class Before
{
    someString;

    before setSomeString {
        int a = "";
    }
    
    public void foo() {
    }
}
