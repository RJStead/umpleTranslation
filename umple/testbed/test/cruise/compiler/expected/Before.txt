Before.ump:8: error: incompatible types: String cannot be converted to int
    int a = "";
            ^
1 error